data/human/names.txt

```diff
@@ -2560,6 +2560,7 @@ phrase "standalone civilian ship names"
 		"Shenandoah"
 		"Sherwood"
 		"Shine in the Black"
+		"Shippy McShipface"
```

Comment:
I think it would probably be better _not_ to have this as a randomly picked name (because also, now it can show up before the poll).